x-pack/legacy/plugins/apm/typings/elasticsearch.ts

```diff
@@ -81,6 +81,11 @@ declare module 'elasticsearch' {
     value: number | null;
   }
 
+  interface TotalValue {
+    value: number;
+    relation: string;
```

Comment:
The relation can be more strictly typed than `string`. It's always `'eq' | 'gte'`:

```
relation: Indicates whether the number of documents returned. Returned values are:

    eq: Accurate
    gte: Lower bound, including returned documents
```

https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html